src/key-binder.c

```diff
@@ -28,15 +29,6 @@ struct _BriskKeyBinderClass {
 };
 
 /**
- * BriskKeyBinder is used to bind global x11 shortcuts
```

Comment:
True. I've restored this, and instead of storing `shortcut` in the key-binder, I think it makes more sense to have key-binder be agnostic to whichever `shortcut` is set, and let the applet handle it.